scripts/jenkins/ardana/manual/generate-input-model.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+source lib.sh
+
+validate_input
```

Comment:
It would be more readable if you would refactor the common lines in the scripts into a separate file. This wouldn't affect many lines here, but I think it would enhance the readability.

Reply:
isnt the `lib.sh` above already such a common file? or do you mean the 4 function calls could be a single call instead?

Reply:
Yes it is, but I was looking more for something to collect the steps. The ardana scripts are all pretty similar and if we refactored the common steps, maintaining (and understanding) them would be easier.

Reply:
Yes. I thought about refactoring and then making some sort of call based on a conditional or so. 
But i opted for the brute force simple way to decouple the scripts. In other words, i did not want to break what was already working.

Reply:
Ok, good point. Better be conservative.